srcalc/src/main/java/gov/va/med/srcalc/vista/ReferenceNote.java

```diff
@@ -0,0 +1,65 @@
+package gov.va.med.srcalc.vista;
```

Comment:
Remember, the `vista` package is for VistA integration. This class belongs in the domain model.